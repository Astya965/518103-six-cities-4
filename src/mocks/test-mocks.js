export const testOffers = [
  {
    id: `1`,
    isPremium: true,
    isFavourite: true,
    previewImage: `img/apartment-01.jpg`,
    price: 55,
    rating: 4.8,
    type: `Apartment`,
    title: `Test name-1`,
    images: [`img/apartment-01.jpg`, `img/apartment-02.jpg`],
    bedrooms: 2,
    maxAdults: 3,
    goods: [`Heating`, `Kitchen`, `Frigde`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      avatarUrl: `img/avatar-max.jpg`,
      id: `host-1`,
      isPro: true,
      name: `Maximum`,
    },
    description: [`A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`, `The building is green and from 18th century.`],
  },
  {
    id: `2`,
    isPremium: false,
    isFavourite: false,
    previewImage: `img/apartment-02.jpg`,
    price: 250,
    rating: 2,
    type: `Room`,
    title: `Test name-2`,
    images: [`img/apartment-02.jpg`],
    bedrooms: 1,
    maxAdults: 1,
    goods: [`Heating`, `Kitchen`, `Frigde`],
    host: {
      avatarUrl: `img/avatar-angelina.jpg`,
      id: `host-3`,
      isPro: false,
      name: `Ellen`,
    },
    description: [`The house itself is surrounded by a gorgeous garden with many hidden lights that make the garden come to life at night.`],
  }
];

export const testOffer = {
  id: `1`,
  isPremium: true,
  isFavourite: true,
  previewImage: `img/apartment-01.jpg`,
  price: 55,
  rating: 4.8,
  type: `Apartment`,
  title: `Test name-1`,
  images: [`img/apartment-01.jpg`, `img/apartment-02.jpg`],
  bedrooms: 2,
  maxAdults: 3,
  goods: [`Heating`, `Kitchen`, `Frigde`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  host: {
    avatarUrl: `img/avatar-max.jpg`,
    id: `host-1`,
    isPro: true,
    name: `Maximum`,
  },
  description: [`A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`, `The building is green and from 18th century.`],
};
