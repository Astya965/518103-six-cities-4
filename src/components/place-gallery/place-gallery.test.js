import React from "react";
import renderer from "react-test-renderer";

import PlaceGallery from "./place-gallery.jsx";
import {testOffer} from "../../mocks/test-mocks.js";

it(`Render PlaceGallery`, () => {
  const tree = renderer.
    create(
        <PlaceGallery images={testOffer.images}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
