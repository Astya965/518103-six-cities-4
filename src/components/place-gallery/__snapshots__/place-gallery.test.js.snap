// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlaceGallery 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Offer photo"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Offer photo"
        className="property__image"
        src="img/apartment-02.jpg"
      />
    </div>
  </div>
</div>
`;
